streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/TableSourceNode.java

```diff
@@ -30,7 +29,7 @@
  */
 public class TableSourceNode<K, V> extends StreamSourceNode<K, V> {
 
-    private StoreBuilder<KeyValueStore<K, V>> storeBuilder;
+    private StoreBuilder storeBuilder;
```

Comment:
Would it be better to put a generic `<S>` her and change to `TableSourceNode<K, V, S extends StateStore>`? Also, should we take `builder.windowedTable()` into account already? Pretty sure the KIP will be accepted.

Reply:
Ack, required some other minor changes, you'll have to let me know what you think